MyMovies/View Controllers/MovieSearchTableViewController.swift

```diff
@@ -27,12 +28,12 @@ class MovieSearchTableViewController: UITableViewController {
     }
     
     override func viewWillDisappear(_ animated: Bool) {
-        if let indexPaths = tableView.indexPathsForSelectedRows {
-            for indexPath in indexPaths {
-                let movieDBMovie = movieController.searchedMovies[indexPath.row]
-                // TODO: Save this movie representation as a managed object in Core Data
-            }
-        }
+//        if let indexPaths = tableView.indexPathsForSelectedRows {
+//            for indexPath in indexPaths {
+//                let movieDBMovie = movieController.searchedMovies[indexPath.row]
+//                 //TODO: Save this movie representation as a managed object in Core Data
+//            }
+//        }
```

Comment:
So the way you are saving the movie to core data works great, the instructions did want you to implement the save here. You could just put the save here and keep everything else in your didSelect method.